.github/workflows/integration_test_run.yaml

```diff
@@ -318,3 +350,15 @@ jobs:
           target: ${{ inputs.zap-target-protocol }}://${{ env.ZAP_TARGET }}:${{ inputs.zap-target-port }}/
           cmd_options: ${{ inputs.zap-cmd-options }}
           rules_file_name: ${{ inputs.zap-rules-file-name }}
+      - name: Save image names
```

Comment:
Can you move these two steps up to the integration_test.yaml to prevent this to be uploaded multiple times? I know this kind of approach caused issues in the past when there are parallel executions

Reply:
Moved, thanks!